docs/modules/behaviors/DATA_BEHAVIOR.md

```diff
@@ -0,0 +1,783 @@
+# Torso Data Behavior
+
+[API Section](#description-of-all-options)
+
+## Goals:
+* This behavior implements simplified interaction with data sources (i.e. TorsoCollection).
+* This behavior manages re-rendering when data changes and automatically adding the returned data to the view's context.
+* This behavior also manages dependencies between data and id containers to allow intelligent re-fetching when ids changes.
+* Provide a clean and concise declaration of how to retrieve data and how to identify the data to retrieve (see [backbone-torso issue #282 on github](https://github.com/vecnatechnologies/backbone-torso/issues/282) for more details).
+
+Binding views to data sources is a task that increases in complexity when the views require data that depends on each other.  This behavior is meant to address that complexity and provide a simple and clean API to define relationships between models in the context of a view.  This Data Behavior removes the need to manage private collection dependencies manually and instead focus on defining the relationship between the models.
+
+Example without the behavior:
```

Comment:
Make the "Example without the behavior" and "Example with the behavior" bold and on their on line.